mongodbatlas/resource_mongodbatlas_cluster.go

```diff
@@ -875,7 +869,7 @@ func expandProviderSetting(d *schema.ResourceData) matlas.ProviderSettings {
 		}
 
 		providerSettings.EncryptEBSVolume = pointy.Bool(true)
-		if encryptEBSVolume, ok := d.GetOkExists("provider_encrypt_ebs_volume"); ok {
+		if encryptEBSVolume, ok := d.GetOk("provider_encrypt_ebs_volume"); ok {
```

Comment:
is there any Default value for this attribute? if so we can add it in order to avoid unwanted false when this value is not set.

Reply:
the default value is set before the If statement, so it can't have a default attribute to due to it's a computed.